code/modules/mob/living/simple_animal/abnormality/waw/nosferatu.dm

```diff
@@ -0,0 +1,226 @@
+				AdjustThirst(30)
+			else
+				AdjustThirst(max((B.bloodiness**2)/800,1))
+		qdel(B)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/nosferatu/proc/AdjustThirst(thirst)
+	blood = clamp(thirst + blood, 0, 400)
+	if(blood > 300)
+		Berzerk()
+
+/mob/living/simple_animal/hostile/abnormality/nosferatu/proc/Berzerk()
+	AdjustThirst(-300)
+	animate(src, transform = matrix()*1.2, color = "#FF0000", time = 10)
+	playsound(get_turf(src), 'sound/abnormalities/nosferatu/transform.ogg', 35, 8)
+	thirst_cooldown = clamp(thirst_cooldown - 1, 0, 4)
+	src.adjustBruteLoss(-300)
+
+//special attacks
+/mob/living/simple_animal/hostile/abnormality/nosferatu/OpenFire()
+	if(!can_act)
+		return
+	if((banquet_cooldown < world.time))
+		Banquet()
+		return
+
+/mob/living/simple_animal/hostile/abnormality/nosferatu/AttackingTarget() //yes, it has a combo system
+	if(!ishuman(target))
+		return ..()
+	var/mob/living/carbon/human/H = target
+	if(thirst <= 0)
+		thirst = thirst_cooldown
+		H.apply_damage(45, BLACK_DAMAGE, null, H.run_armor_check(null, BLACK_DAMAGE))
+		playsound(get_turf(src), 'sound/abnormalities/nosferatu/bat_attack.ogg', 50, 1)
+		to_chat(target,"<span class='danger'>The [src] attacks you savagely!</span>")
+		AdjustThirst(40)
+	else
+		thirst -= 1
+	AdjustThirst(40)
+	if(H.health < 0 || H.stat == DEAD)
+		H.Drain()
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/nosferatu/proc/BatSpawn() //stolen from titania
+	playsound(get_turf(src), 'sound/abnormalities/nosferatu/batspawn.ogg', 50, 1)
+	//How many we have spawned
+	listclearnulls(spawned_bats)
+	for(var/mob/living/L in spawned_bats)
+		if(L.stat == DEAD)
+			spawned_bats -= L
+	if(length(spawned_bats) >= bat_spawn_limit)
+		return
+
+	//Actually spawning them
+	for(var/i=bat_spawn_number, i>=0, i--)	//This counts down.
+		var/mob/living/simple_animal/hostile/nosferatu_mob/B = new(get_turf(src))
+		spawned_bats+=B
+	addtimer(CALLBACK(src, .proc/BatSpawn), summon_cooldown_time)
+
+/mob/living/simple_animal/hostile/abnormality/nosferatu/proc/Banquet()
+	banquet_cooldown = world.time + banquet_cooldown_time
+	can_act = FALSE
+	for(var/turf/L in view(2, src))
+		var/obj/effect/temp_visual/cult/sparks/S = new /obj/effect/temp_visual/cult/sparks(L)
+	playsound(get_turf(src), 'sound/abnormalities/nosferatu/special_start.ogg', 50, 0, 5)
+	SLEEP_CHECK_DEATH(10)
+	for(var/turf/T in view(2, src))
+		var/obj/effect/temp_visual/smash_effect/bloodeffect =  new(T)
+		bloodeffect.color = "#b52e19"
+		for(var/mob/living/L in T)
+			if(faction_check_mob(L))
+				continue
+			L.apply_damage(banquet_damage, BLACK_DAMAGE, null, L.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
+			if(L.health < 0 && ishuman(L))
+				var/mob/living/carbon/human/H = L
+				H.Drain()
+	playsound(get_turf(src), 'sound/abnormalities/nosferatu/attack_special.ogg', 50, 0, 5)
+	SLEEP_CHECK_DEATH(3)
+	can_act = TRUE
+
+//spawned mobs
+/mob/living/simple_animal/hostile/nosferatu_mob
+	name = "Sanguine bat"
```

Comment:
```suggestion
	name = "sanguine bat"
```
I figure this is probably an improper noun? If it's supposed to be capitalized, you can also go for
```suggestion
	name = "\improper Sanguine bat"
```
instead.